docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -12,16 +12,22 @@ You can use UID2 server-side SDKs to facilitate decrypting of UID2 advertising t
 <!-- This guide includes the following information:
 
 - [Overview](#overview)
+- [Audience](#audience)
 - [Initialization](#initialization)
 - [Interface](#interface)
   - [Response Content](#response-content)
   - [Response Statuses](#response-statuses)
-* [FAQs](#faqs) -->
+* [FAQs](#faqs)
+* [Usage for UID2 Sharers](#usage-for-uid2-sharers) -->
```

Comment:
should we add this for c#/c++/python ref guides?

Reply:
Done -- if I understood correctly -- but let's discuss.